include/xrpl/server/detail/PlainHTTPPeer.h

```diff
@@ -105,7 +105,7 @@ PlainHTTPPeer<Handler>::run()
 {
     if (!this->handler_.onAccept(this->session(), this->remote_address_))
     {
-        boost::asio::spawn(
+        ripple::util::spawn(
```

Comment:
```suggestion
        util::spawn(
```

`ripple::` can be removed